assets/app/view/game_page.rb

```diff
@@ -407,7 +407,11 @@ def render_action
       when Engine::Round::Auction
         h(Game::Round::Auction, game: @game, user: @user)
       when Engine::Round::Merger
-        h(Game::Round::Merger, game: @game)
+        if current_entity_actions.include?('buy_train')
```

Comment:
why do you need this? is this gonna break any other game?

Reply:
I need to buy a train during merge round. If someone else had this action it should have shown something. But without that nothing is displayed. 

We can add the buy train view to merge as well, but I think showing the map etc is better and thats the easiest way